smk/gene_annotation.smk

```diff
@@ -337,7 +337,9 @@ rule gene_annot_kofamscan:
         remote_dir=$(dirname {output})
         time (
             exec_annotation -k {input.ko_list} -p {input.prok_hal} --tmp-dir tmp -f mapper-one-line --cpu {threads} -o kofam_mapper.txt {input.faa}
-            {script_dir}/kofam_hits.pl --pathway-map {input.pathway_map} --module-map {input.module_map} kofam_mapper.txt > {output}
+            echo -e "#Definitions downloaded\t$(stat -c '%y' {minto_dir}/data/kofam_db/ko_list | cut -d' ' -f 1)\t$(stat -c '%y' {input.module_map} | cut -d' ' -f 1)\t$(stat -c '%y' {input.pathway_map} | cut -d' ' -f 1)" > kofam_processed.txt
```

Comment:
Minor detail: Within the shell mode, it is better to encode tab as `\\t` since it will now be literally converted to tab-character before passing to shell. If you do a `-p` and copy the command, it will not work since tab won't be copied correctly. Sending `\\t` will pass `\t` to the shell.
The command will work and output will not change even as it is. It's purely for debugging purposes etc...

Reply:
Also, could we replace `{minto_dir}/data/kofam_db/ko_list` with `{input.ko_list}`? Btw, I like this approach to use `stat` on `{input}`s rather than file-paths generated inside the rule. Could we also define the files that we `stat` as part of `input`?

Reply:
Annotation databases are now rule inputs [325d61a](https://github.com/arumugamlab/MIntO/pull/41/commits/325d61ae70eb12a90e5d4f564c0c90450000b07c)